meta/headers.go

```diff
@@ -0,0 +1,60 @@
+package meta
+
+import (
+	"sort"
+)
+
+// Header is used to manage the encoding and decoding of csv formatted files by using their column headers.
+// This allows adding or removing columns in the file without breaking existing applications that may not
+// understand or expect extra columns.
```

Comment:
Is this feature covered in the test case? From what I can see it's just testing difference in order.

Reply:
Doesn't look like it. Will add one.

Reply:
That took longer, than I thought.  Have to swap one of the maps around and add an existence check to avoid overwriting the default (0) map values if not found.